app/views/candidate_mailer/find_has_opened.erb

```diff
@@ -0,0 +1,20 @@
+<% if @application_form.first_name.present? %>
+  Dear <%= @application_form.first_name %>,
+<% end %>
+
+You can now find teacher training courses starting in the <%= @academic_year %> academic year.
+
+Find your course and get your application ready:
+
+https://www.gov.uk/find-postgraduate-teacher-training-courses
```

Comment:
Any reason not to send candidates straight to Find?

Could/should we also set some kind of query string so that we can track clicks from email?

eg

```suggestion
https://www.find-postgraduate-teacher-training.service.gov.uk?utm_source=findopen&utm_medium=email
```

Reply:
Yea i did think about that, but assumed it was due to there being additional content on the start page that my be relevant that they haven't seen?

I know it's been updated a couple of times in the last cycle

Reply:
I kinda doubt that many people read the text beneath the "Start now" button to be honest...

Feel like we could afford to skip the start page here? Especially as it'd mean we could measure email performance. Any thoughts @EmmaFrith?

Reply:
Have spoken to Emma and we're going to link to Find